tests/run-tests.py

```diff
@@ -910,6 +915,11 @@ def main():
         action="store_true",
         help="delete the .exp and .out files from failed tests and exit",
     )
+    cmd_parser.add_argument(
+        "--run-failed",
```

Comment:
maybe the option can be called `--run-failures`, to match the names of the other two options above

Reply:
Ok